README.md

```diff
@@ -29,6 +29,22 @@ The rendered final report is located [here](https://github.com/UBC-MDS/olympic_m
 
 ## Usage
 
+#### 1\. Using Docker
+
+Please install [Docker](https://www.docker.com/get-started) if you have not already done so.  Then clone this repository and run the following command at the command line/terminal from the root directory of this project:
+
+```bash
+docker run --rm --platform linux/amd64 -v $(PWD):/home/data_analysis squisty/olympic_medal_htest:latest make -C /home/data_analysis
```

Comment:
This is probably the last thing we would change but I think its best to link to a specific tagged image rather than :latest as currently our action workflow will update that latest release whenever a PR or push is made.

Reply:
Sorry.  So instead of ".latest", what should it be specifically?